libraries/c_sdk/standard/mqtt/src/iot_mqtt_serializer_deserializer_wrapper.c

```diff
@@ -765,6 +768,37 @@ IotMqttError_t _IotMqtt_pubackSerializeWrapper( uint16_t packetIdentifier,
 
 /*-----------------------------------------------------------*/
 
+IotMqttError_t _IotMqtt_managedDisconnect( IotMqttConnection_t mqttConnection )
```

Comment:
This function doesn't seems to belong in iot_mqtt_serializer_deserializer_wrapper.c file.